### ISSUE TITLE
New Term Request: in situ hybridization evidence

### ISSUE THREAD

**Issue by srobb1** [evidenceontology/evidenceontology#236]
You currently have:

fluorescence in situ hybridization evidence http://purl.obolibrary.org/obo/ECO_0001047 

It would be super useful to us if there was a parent term of just "in situ hybridization evidence". We do have fluorescence evidence but more often than not, it is not fluorescent, but NBT-BCIP (visual purple) in situ hybridizations. Just a more general parent term would fit our needs.

Thank you,
Sofia

**Comment by jbmunro** [comment]
Hi @srobb1 

Sorry for not getting to this one sooner.

I am happy to create the requested parent term but in general, we try to avoid generalized "catch-all" parent terms and parent terms with only one child.   I was thinking that I could add the requested parent term and your suggested sibling term to 'fluorescence in situ hybridization evidence': NBT-BCIP (visual purple) in situ hybridization.  However, when I started to read up on NBT-BCIP (visual purple) in situ hybridization, the first article I found was this: PMID:17612300, entitled "Fluorescent in situ hybridization employing the conventional NBT/BCIP chromogenic stain".  The abstract then goes on that [NBT-BCIP in situ hybridization] "has significant fluorescence that enables the use of confocal microscopy to generate high-resolution three-dimensional (3-D) imaging of gene expression."

Sorry for the confusion, but it seems to me that the example provided is in fact a type of 'fluorescence in situ hybridization evidence'.  Please correct me if I am wrong.

Are there perhaps additional examples that you could provide for which new terms could be made?

James

**Comment by srobb1** [comment]
Hi James,

Thank you for getting back to me. I understand your requirements for only single siblings. Maybe we can work through this together and figure this out. 

That is interesting, all the years I have been using NBT/BCIP, I did not know it was fluorescent. But I see from this paper that the use of fluorescent visualization with NBT/BCIP is not conventional. :) 

In the planarian research community we use the NBT/BCIP as colorimetric and separate reagents for fluorescent in situ hybridization protocols. The colorimetric can not be used for double labeling, while the fluorescent can. How about

in situ hybridization evidence (parent)
 -- Fluorescent in situ hybridization evidence
 -- Colorimetric in situ hybridization evidence

I don't think the non-fluorescent in situ hybridization evidence term needs to specify NBT/BCIP, just that it doesn't eliminate in situ hybridization evidence that is not visualized with fluorescence. We are putting together a little tool to help annotate gene expression in anatomical location. We are using ontology terms for every part of this. Our annotators will not select 'Fluorescent in situ hybridization evidence' for evidence associate with our purple in situ hybridization evidence. I was hoping to use your ontology for all of our evidence types.

Here is a protocol paper where the two (fluorescent:FISH & colorimetric:WISH) are discussed together as being different. Though this paper does not refer to it as colorimetric.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610298/ 

Thank you for talking this through with me, any changes to help us with our in situ patterns not visualized with fluorescence will be greatly appreciated.

Sofia

**Comment by jbmunro** [comment]
Hi Sofia,

Thanks for the explanation.

Cool! I like the 'colorimetric in situ hybridization evidence' suggestion.  Let me do a little reading and put some suggestions forth to our group for discussion.  I'll get back to you asap.

James

**Comment by srobb1** [comment]
@jbmunro Thank you!

**Comment by jbmunro** [comment]
Hi Sofia,

OK, so I am going to create a new parent term 'in situ hybridization evidence' and place 'fluorescence in situ hybridization evidence' and the new term 'colorimetric in situ hybridization evidence' as children.  I hope to push this to GitHub early next week, if not sooner.  (I will keep you posted.)  The new terms will be made available with the next release.

May I ask what organization you are associated with?  I would like to note this with the new terms.

More later.

James

**Comment by srobb1** [comment]
Great! I am with the Stowers Institute of Medical Research (SIMR). This is specifically to help out with our planarian anatomy ontology (PLANA). I also work with the planarian phenotype ontology (PLANP) and help out a smidge with the UPheno project. And I am have a couple other anatomy ontologies for organisms used at SIMR.

**Comment by jbmunro** [comment]
Hi Sofia,

I pushed the new terms - eco-edit.owl and pasted them below.  They will be more readily available with the next release.  

Thanks for the info.  I will update the term requester information at some point in the future.

Closing the ticket.  

James

ECO:0001836
in situ hybridization evidence
A type of expression pattern evidence in which single-stranded DNA or RNA (probe) are annealed to complementary DNA or RNA in a portion or section of tissue. 

ECO:0001837
in situ hybridization evidence used in automatic assertion
A type of in situ hybridization evidence that is used in an automatic assertion.

ECO:0001838
in situ hybridization evidence used in manual assertion
A type of in situ hybridization evidence that is used in a manual assertion.

ECO:0001839
colorimetric in situ hybridization evidence
A type of nucleic acid localization evidence and in situ hybridization evidence resulting from the use of colorimetric probes to detect complementary sequences of nucleic acids.
PMID:23497040

ECO:0001840
colorimetric in situ hybridization evidence used in automatic assertion
A type of colorimetric in situ hybridization evidence that is used in an automatic assertion.

ECO:0001841
colorimetric in situ hybridization evidence used in manual assertion
A type of colorimetric in situ hybridization evidence that is used in a manual assertion.
